tools/scoring/score.py

```diff
@@ -424,21 +424,35 @@ def score(division,
         data_new['unsolved'] = data_new.num_check_sat - data_new.correct
     # Set correct/error column for solved benchmarks.
     else:
-        # Filter job pairs based on given verdict. For the sat/unsat scoring
-        # only satisfiable/unsatisfiable instances are considered, i.e., if
-        # either the expected status is sat/unsat or a solver solves the
-        # instance.
         if filter_result:
-            expected = set([filter_result, RESULT_UNKNOWN])
+            if filter_result == RESULT_UNSAT:
+                negated_filter_result = RESULT_SAT
+            else:
+                negated_filter_result = RESULT_UNSAT
```

Comment:
it'd be good to inline

Reply:
Do you mean this: `negated_filter_result = RESULT_SAT if filter_result = RESULT_UNSAT else RESULT_UNSAT`?

I found it a bit confusing and the line doesn't fit...

Reply:
That's what I meant, yes. But if you think it's confusing I'd say it's fine as is.

Reply:
I also prefer it this way, and would maybe add an `assert filter_result == RESULT_SAT` in the `else` branch.